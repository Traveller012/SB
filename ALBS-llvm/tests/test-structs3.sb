{ : int} main = [ 

    struct mystruct1 l1;
    struct mystruct2 l2; 

    l1.a = 100;
    l1.b = 200.2;

    l2.d = 'a';
    l2.e = 5.2;
    l2.x = 2;

    print(l1.a);
    print(l1.b);
    print(l2.d);
    print(l2.e);
    print(l2.x);
     

	rtn 0;
]

struct mystruct1[

	int a;
	flt b;
 	
]

struct mystruct2[

    chr d;
    flt e;
    int x; 
    
]
