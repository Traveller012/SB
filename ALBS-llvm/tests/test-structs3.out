100
200.200000
a
5.200000
2